README.md

```diff
@@ -1,34 +1,58 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
-## Getting Started
-
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
-
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
-
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
-
-## Learn More
-
-To learn more about Next.js, take a look at the following resources:
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+<h1 align="center">
+  <img src="./src/assets/brand.png" alt="Logo Conduit"/>
+  <br>
+  Conduit App
+  <h3 align="center"><a href="https://conduit-front.netlify.app/">Project Demo</a></h3>
+  <h3 align="center"><a href="https://conduit-storybook.netlify.app/">Storybook Demo</a></h3>
+</h1>
+
+## 📝 About the project
+Conduit is the name of the project of the [Real World App](https://github.com/gothinkster/realworld).<br>
```

Comment:
O que acha de:
"Conduit is a "Medium" clone based on the Real World App](https://github.com/gothinkster/realworld) spec.

Reply:
LGTM! I fix both of request changes in this commit [190dbb](https://github.com/joaooliveiradev/conduit-frontend/pull/12/commits/190dbb6ba6f34e7229071819ad1a71ee4998c47c)